frontend/src/components/VRecentSearches/VRecentSearches.vue

```diff
@@ -37,20 +37,37 @@
         :id="`option-${idx}`"
         :key="idx"
         role="option"
-        class="description-regular my-1 rounded-sm border-1.5 p-2 hover:bg-dark-charcoal-10"
+        class="group/entry description-regular flex h-10 flex-row items-center gap-2 rounded-sm border-1.5 pe-1 ps-2 hover:bg-dark-charcoal-10"
         :class="idx === selectedIdx ? 'border-pink' : 'border-tx'"
         :aria-selected="idx === selectedIdx"
         @click="handleClick(idx)"
       >
+        <VIcon name="search" />
         {{ entry }}
+        <VIconButton
+          variant="transparent-gray"
+          :icon-props="{ name: 'close-small' }"
+          size="small"
+          :label="$t('recentSearches.clearSingle.label', { entry }).toString()"
+          class="ms-auto group-hover/entry:flex"
+          :class="{ hidden: bordered }"
+          @click.stop="handleClearSingle(idx)"
+        />
       </li>
       <!-- eslint-enable -->
     </ul>
-    <span v-else class="description-regular mx-2 my-3">
+    <span
+      v-else
+      class="description-regular flex h-10 flex-row items-center ps-3"
+      :class="{ 'pe-1': !bordered }"
+    >
       {{ $t("recentSearches.none") }}
     </span>
 
-    <span class="caption-regular mx-2 my-3 text-dark-charcoal-70">
+    <span
+      class="caption-regular mt-auto flex flex-row items-center px-3 pt-3 text-dark-charcoal-70"
```

Comment:
Thanks for catching this. Since the disclaimer can span more than one line, limiting the height to `h-10` can cause issues. fc15ab9 uses a padding based approach that can handle both single line (for larger screens) or multiple lines (for mobile screens).

Reply:
As for `p-3`, it was derived from this calculation:

Fixed height = 2.5 rem
Height of single line = ~1 rem
Padding = (2.5-1)/2 rem = 0.75rem

Reply:
Oh, you are very right. The change looks good now